types/evidence_test.go

```diff
@@ -246,7 +247,7 @@ func makeHeaderRandom() *Header {
 	return &Header{
 		Version:            version.Consensus{Block: version.BlockProtocol, App: 1},
 		ChainID:            tmrand.Str(12),
-		Height:             int64(tmrand.Uint16()) + 1,
+		Height:             int64(mrand.Uint32()&(1<<16-1)) + 1,
```

Comment:
nit: I think having `mrand.Uint32() + 1` is fine enough i.e. it's not going to overflow which I believe was the reason for having it uint16 to begin with

Reply:
Yeah, I think so too. I just didn't want to change any logic to make this PR as uncontroversial as possible. I'll update this to `mrand.Uint32() + 1` shortly.

Reply:
https://github.com/tendermint/tendermint/pull/6364/commits/ad4bfd2d6249c123f4c2b3a59e7604800559c7e9